app/models/tiebreaker.rb

```diff
@@ -0,0 +1,13 @@
+class Tiebreaker < ActiveRecord::Base
+  enum tiebreaker: [:most_points, :most_weak_side_wins, :fewest_played]
+  validates_uniqueness_of :tiebreaker
+
+  include RankedModel
+  ranks :ordinal
+  scope :ordered, -> { rank(:ordinal) }
+
+  def self.available_tiebreakers
+    taken = pluck(:tiebreaker)
+    tiebreakers.reject { |k, v| taken.include?(v) }
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.